getting-started/README.adoc

```diff
@@ -99,13 +99,15 @@ endif::[]
 .Kafka instance configuration details
 image::sak-configure-kafka-instance.png[Image of instance configuration details in Create Kafka instance window]
 
-The new Kafka instance is listed in the instances table. When the instance *Status* is *Ready*, you can start using the Kafka instance. You can use the options icon (three vertical dots) to view instance details, connect to the instance, or delete the instance.
+The new Kafka instance is listed in the instances table. In some cases, you might need to wait several minutes for the instance creation process to complete.
```

Comment:
Suggest "...you might need to wait several minutes for the instance creation process to _finish_".

Per IBM style:
"Use _complete_ only as a transitive verb. For example, write "the program completes the operation successfully", not "the program completes".

Reply:
Great catch! Absolutely, fixed. Thank you!